distribution/ddf-common/src/main/resources-filtered/etc/custom.system.properties

```diff
@@ -95,6 +95,8 @@ solr.username=admin
 # Comma-separated list of metacard types that do not support optimistic updates.
 solr.commit.nrt.metacardTypes=workspace,metacard.query,metacard.list,query-template,attribute-group,resource-note
 
+migration.supported.versions = 2.13.9
```

Comment:
In the case above, if 2.13.9 (the ddf exported from) has migration.supported.versions = 2.13.8, will the ddf that does the import (say 2.14.x) now have migration.supported.versions = 2.13.8 in custom.system.properties?  If so, will that cause any confusion or problems?

Reply:
Yes that's a fair point. I can add this system property to the "preserved" list, so that a migration import will not overwrite this in the importing system.